template_content/smart_contracts/{% if deployment_language == 'python' %}config.py{% endif %}

```diff
@@ -21,14 +21,15 @@ logger = logging.getLogger(__name__)
 # define contracts to build and/or deploy
 contracts = [helloworld.app]
 
+
 # define deployment behaviour based on supplied app spec
 def deploy(
     algod_client: AlgodClient,
     indexer_client: IndexerClient,
     app_spec: ApplicationSpecification,
     deployer: Account,
 ) -> None:
-    is_local = is_localnet(algod_client)
+    is_main = is_mainnet(algod_client)
```

Comment:
Also change this to `is_mainnet`, will need to change the import as well and use `algokit_utils.is_mainnet`